### PR TITLE
Manage members fix

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
This PR removes the ability to remove yourself from the channel info manage members screen.

Submitted on behalf of @jarredwitt

### CODE REVIEW

File: app/scenes/channel_members/channel_members_container.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ import {getProfilesInChannel} from 'service/actions/users';
 import ChannelMembers from './channel_members';
 
 function mapStateToProps(state) {
-    const currentChannelMemberCount = getCurrentChannelStats(state) && getCurrentChannelStats(state).member_count;
+    // Decrease the member count since we exclude the current user.
+    const currentChannelMemberCount = getCurrentChannelStats(state) && getCurrentChannelStats(state).member_count - 1;
```
Comment (enahum) [inline app/scenes/channel_members/channel_members_container.js]: What if we show the correct member count but when checking if we need to load more we subtract the one

**Comment by jarredwitt** [comment]
This is blocked by the More Channels PR since there were some changes made when moving the ```member_list``` to ```custom_list```.

**Comment by enahum** [comment]
added the label waiting for #276 to be merged

**Comment by enahum** [comment]
ok @jarredwitt you can now update this one

**Comment by jarredwitt** [comment]
Should be ready to go. I added a ```disableSelect``` property to the member_row and made it to where the current user is shown but can't be selected. 

simulator screen shot feb 20 2017 2 03 47 pm

**Pull request merged**